consensus/istanbul/uptime/config.go

```diff
@@ -24,32 +24,38 @@ const (
 // ComputeLookbackWindow computes the lookbackWindow based on different required parameters.
 // getLookbackWindow represents the way to obtain lookbackWindow from the smart contract
 func ComputeLookbackWindow(epochSize uint64, defaultLookbackWindow uint64, isDonut bool, getLookbackWindow func() (uint64, error)) uint64 {
-	if !isDonut {
-		return defaultLookbackWindow
-	}
+	var returnValue uint64
 
 	if epochSize <= istanbul.MinEpochSize {
 		panic("Invalid epoch value")
 	}
 
-	value, err := getLookbackWindow()
-	if err != nil {
-		value = MinSafeLookbackWindow
+	if !isDonut {
```

Comment:
I usually prefer to flip the then/else to have `if isDonut {` instead, except when there is a reason not to.  Having the shorter block first is nice, but not necessarily worth it.